lib/rules/focus-order-semantics.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "focus-order-semantics",
```

Comment:
You'll need to add integration tests for this rule. Each rule should have integration tests.

Reply:
Done.